core-java-8/src/main/java/com/baeldung/bifunction/Calculator.java

```diff
@@ -0,0 +1,28 @@
+package com.baeldung.bifunction;
+
+import java.util.function.BiFunction;
+
+public class Calculator {
+	
+    private Integer a = 0;
+    private Integer b = 0;
+	
+    public Calculator(Integer a, Integer b) {
+        this.a = a;
+        this.b = b;
+    }
+
+    public Integer calculate(Integer a, Integer b, BiFunction<Integer, Integer, Integer> obj){

```

Comment:
I can't see what this function adds to `Calculator`. It doesn't use the members `a` or `b`. It could, therefore, be static. It's doing nothing more than the `apply` function on the method you pass in, which makes it a pretty redundant thing.